db/1.13/modules/ROOT/pages/database-configure-usebulkcopy.adoc

```diff
@@ -0,0 +1,22 @@
+= Configure Use Bulk Copy for Database Connector SQL Server Connection
+
+ You can configure the bulk copy for batch insert in Anypoint Connector for Database (Database Connector) for the Microsoft SQL Server Connection. This feature allows you to enable the driver to do bulk copy operations when executing batch insert operations. Configure this feature as a connection property in the Database Connector Configuration screen. 
+
+ . In Studio, navigate to the *Global Elements* tab.
+ . Click *Create*.
+ . In the search bar, type `database`.
+ . Select *Database config*.
+ . Click *OK*.
+ . In the *Global Elements Properties* screen, set *Connection* to *Microsoft SQL Server Connection*.
+ . Click the *Advanced* tab.
+ . Set *Connection properties* to *Edit inline*.
+ . Click the plus sign to add a new property.
+ . In the *Connection property* screen, set *Key* to `useBulkCopyforBatchInsert`.
+ . Set *Value* to `true`.
+
+ image::database-bulkcopy.png[useBulkCopyforBatchInsert property created under the Connection property section]
```

Comment:
This line and the "See Also" heading are appearing as if they're formatted as code.